packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -37,6 +43,14 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
           .load({service: 'jiraServer', orgTeamIds, teamIds: [teamId]})
         return providers
       }
+    },
+    projects: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(JiraServerRemoteProject))),
+      description:
+        'A list of projects accessible by this team member. empty if viewer is not the user',
+      resolve: async ({teamId, userId}, _args: unknown, {dataLoader}) => {
+        return fetchJiraServerProjects(teamId, userId, dataLoader)
```

Comment:
-1 `fetchJiraServerProjects` looks like a dataloader, especially considering how expensive it might be. 
perhaps refactor into a `jiraServerProjects` dataloader?